File: com.getshop.messages/src/com/thundashop/core/storemanager/data/Store.java
package com.thundashop.core.storemanager.data;

import com.thundashop.core.common.DataCommon;
import com.thundashop.core.usermanager.data.User;
import java.util.List;

/**
 *
 * @author ktonder
 */
public class Store extends DataCommon {
    public String webAddress;
    public String webAddressPrimary;
    public List<String> additionalDomainNames;
    public boolean registeredDomain=false;
    public boolean readIntroduction=false;
    public boolean isExtendedMode=true;
    public StoreConfiguration configuration;
    public boolean isVIS = false; //Very important shop.
    public boolean isDeepFreezed = false;
    public String deepFreezePassword = "";
    
    /**
     * This specifies if this store is setup as a template
     * If it is a template it can be cloned into another store :D
     */
    public boolean isTemplate = false;
    public User registrationUser;
}

File: com.getshop.client/events/core/storemanager/data/Store.php
<?php
class core_storemanager_data_Store extends core_common_DataCommon  {
	/** @var String */
	public $webAddress;

	/** @var String */
	public $webAddressPrimary;

	/** @var String */
	public $additionalDomainNames;

	/** @var String */
	public $registeredDomain;

	/** @var String */
	public $readIntroduction;

	/** @var String */
	public $isExtendedMode;

	/** @var core_storemanager_data_StoreConfiguration */
	public $configuration;

	/** @var String */
	public $isVIS;

	/** @var String */
	public $isDeepFreezed;

	/** @var String */
	public $deepFreezePassword;

	/** @var String */
	public $isTemplate;

	/** @var core_usermanager_data_User */
	public $registrationUser;

}
?>